apps/web/src/utils/url.ts

```diff
@@ -43,3 +43,14 @@ export const getOriginPath = (url: string): string => {
     return url
   }
 }
+
+// Function to strip URL parameters, returning only the base URL
+export const stripURLParams = (url: string): string => {
+  try {
+    const urlObj = new URL(url)
+    return `${urlObj.origin}${urlObj.pathname}`
+  } catch (e) {
+    console.error('Error parsing URL', url, e)
```

Comment:
We generally don't use console logging directly. You can either remove it and ignore the error or log it using the `logError` utility.